yapf.ini

```diff
@@ -1,3 +1,3 @@
 [style]
 based_on_style = pep8
-column_limit = 120
+column_limit = 160
```

Comment:
You mean, we've got wider display now? :)

Reply:
Yeah, my display width is ... kidding.

just following gokart and thunderbolt
https://github.com/m3dev/gokart/blob/master/yapf.ini
https://github.com/m3dev/thunderbolt/blob/master/yapf.ini

Reply:
👍